test/tests/outputs/oversample/oversample_file.i

```diff
@@ -42,7 +42,7 @@
 [Outputs]
   [./exodus]
     type = Exodus
-    refinements = 2
+    refinements = 1
```

Comment:
It affects about 10 tests in Marmot and that's it. We talked it over and all think it makes more sense here as well.